bioconda_utils/utils.py

```diff
@@ -639,3 +639,26 @@ def get_list(key):
 
     default_config.update(config)
     return default_config
+
+
+def modified_recipes(git_range, recipe_folder):
```

Comment:
It wasn't working for me locally without `shell=True`. But after your comment about git expanding the glob I upgraded git from 1.9.1 to 2.11 and now it works even with no `shell=True`. Something to consider if/when this is run in `simulate-travis.py` when folks may not necessarily have the latest git.